src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 import { Loader } from '../Loader';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  todo: Todo;
+  closeModel: () => void;
+};
+
+export const TodoModal: React.FC<Props> = ({ todo, closeModel }) => {
+  const [user, setUser] = useState<User | null>(null);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+
+    getUser(todo.userId)
+      .then(setUser)
+      .finally(() => setLoading(false));
+  }, []);
```

Comment:
The dependency array in `useEffect` is empty, which means the effect will run once and never run again, even if the `todo` prop changes. This might be an issue if the parent component changes the `todo` prop after the first render. Consider adding `todo.userId` to the dependency array to ensure that the user data is fetched whenever the `todo` prop changes.